### ISSUE TITLE
Make "multiCommand.commands" an object with commands as keys

### ISSUE THREAD

**Issue by rioj7** [ryuta46/vscode-multi-command#30]
Currently the `"multiCommand.commands"` configuration is an array of objects. If you have this setting in the global `settings.json` and workspace `settings.json` they don't merge.

If I want to use a particular `multiCommand.XYZ` in all my workspaces I have to copy/paste the setting.

If the setting was an object the setting will be merged for global-multi_root_workspace-workspace.
The `command` argument of the objects is now the key of the object in the parent object

``` lang-json
  "multiCommand.commands": {
    "multiCommand.someName1": {
      "sequence": [
            "cursorLeftSelect",
            "editor.action.clipboardCutAction"
      ]
    },
    "multiCommand.someName2": {
      "sequence": [
            "editor.action.clipboardCutAction",
            "cursorLeftSelect"
      ]
    }
  }
```

In the extension it is easy:

* test if the configuration `multiCommand.commands` is an array or object
* enumerate the array elements or
* enumerate the object keys and get the value for that key property.

**Comment by ryuta46** [comment]
As you mentioned, "multiCommand.commands" is an array of object and doesn't merge user settings and workspace settings.
In the next version, I'll change it to the format you suggested, while maintaining backwards compatibility.

**Comment by ryuta46** [comment]
Implemented in 1.5.0.
Please reopen this issue or create new issue if you have any problem about this feature.
Thanks.
